hub/core/index/index.py

```diff
@@ -246,6 +246,11 @@ def validate(self, parent_length: int):
 
         # Check ints that are too large (positive or negative)
         if isinstance(self.value, int):
+            try:
+                if self.value >= parent_length or self.value < -parent_length:
+                    raise ValueError
+            except ValueError:
+                print(f"Index {self.value} out of range for tensor with length {parent_length}")
```

Comment:
This condition is already checked in the lines below. But what is the point of raising a valueerror, catching it, and then printing a message, practically swallowing the exception?

Reply:
Yes you are right there is no need for raising value error.